archiver_test.go

```diff
@@ -74,7 +74,7 @@ func symmetricTest(t *testing.T, name string, ar Archiver) {
 
 		if info.IsDir() {
 			// stat dir instead of read file
-			_, err := os.Stat(origPath)
+			_, err = os.Stat(origPath)
```

Comment:
I see, thanks.

I would want to discuss whether a shadowed variable with name `err` and type `error` is something that warrants issuing warnings about, but that's a discussion to have elsewhere.